catch_analysis_tools/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+from importlib.metadata import version as _version, PackageNotFoundError
```

Comment:
This `__version__` business is only needed in the main `__init__.py` file.  This file may be blank.